src/equations/compressible_navier_stokes_2d.jl

```diff
@@ -204,6 +211,76 @@ end
   return SVector(rho, v1, v2, T)
 end
 
+
+# Convert conservative variables to entropy
+# Note, only w_2, w_3, w_4 are needed for the viscous fluxes so we avoid computing
+# w_1 and simply copy over rho.
```

Comment:
Why? Is `w_1` so much more expensive to compute? I am asking since this seems like... a headache to come, if the output of `cons2entropy` is *not* the entropy variables...

Reply:
it involves logarithms so I was trying to avoid it because we do not need it. The first variable looks like
```
w_1 = (gamma - s) / (gamma - 1) - 0.5 * rho * (v1^2 + v^2) / p
```
where `s = log(p) - gamma*log(rho)`. If we want we can just have this routine call the `cons2entropy` routine from the compressible Euler equations (similar to what is currently done with `cons2prim`). That could avoid confusion.

Reply:
It would be nice to avoid computing w1. Maybe we can revisit this when we address skipping computing gradients for certain variables?

Reply:
Going with the "make it work, then make it fast" mantra, we can use `cons2entropy(u, equations_parabolic) = cons2entropy(u, equations_parabolic.equations_hyperbolic)`